www/js/overlay.js

```diff
@@ -1,7 +1,9 @@
+/* eslint-disable import/no-unresolved */
 const electron = require('electron');
 const h = require('hyperscript');
 const ip = require('ip');
 const copy = require('copy-to-clipboard');
+const { obs: defaultPrefs } = require('./js/defaults.json');
```

Comment:
is this path correct? this file is in the same directory as `defaults.json`

Reply:
Yes, the paths are referenced from `www` folder. So `./defaults.json` would give an error.